modules/cucumber_testsuite/main.tf

```
@@ -30,12 +29,15 @@ locals {
   host_key => lookup(var.host_settings[host_key], "mac", null) if var.host_settings[host_key] != null }
   additional_repos = { for host_key in local.hosts :
   host_key => lookup(var.host_settings[host_key], "additional_repos", {}) if var.host_settings[host_key] != null }
+  images = { for host_key in local.hosts :
+  host_key => lookup(var.host_settings[host_key], "image", null) if var.host_settings[host_key] != null }
 }
 
 module "srv" {
   source                         = "../libvirt/suse_manager"
   base_configuration             = module.base.configuration
   product_version                = var.product_version
+  image                          = lookup(local.images, "srv", null)
```

If the image is not defined we should use the default value for the variable which is: "default".
```suggestion
  image                          = lookup(local.images, "srv", "default")
```